packages/amplify-gen2-codegen/src/storage/source_builder.ts

```diff
@@ -48,15 +54,22 @@ export const renderStorage = (storageParams: StorageRenderParameters = {}) => {
       ),
     );
   }
-  if (storageParams.triggers) {
+  if (storageParams.triggers && Object.keys(storageParams.triggers).length) {
```

Comment:
nit: Store `storageParams.triggers` in a variable for repeated access.

Reply:
Done [4559ff3](https://github.com/aws-amplify/amplify-cli/pull/13928/commits/4559ff3b5586e6f88ec42465d36789570e396429)